index.html

```diff
@@ -0,0 +1,128 @@
+            <li><a href="about.html">About</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h2 class="h2-big">LET'S LEARN CODING TOGETHER</h2>
+      <p>Platform that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone. </p>
+      <article>
+        <h2 class="date">2022.01.05(WED) ~ 30(SAT)</h2>
+        <p class="place">@ Addis Ababa, Ethiopia, Africa.</p>
+      </article>
+    </section>
+  </main>
+</div>
+  <section class="section-2">
+    <h2>Main Program</h2>
+    <hr>
+    <div class="section-2-container"> 
+    <article>
+      <i class="far fa-podium fa-2x"></i>
+      <h2 class="org">Lecture</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="fab fa-wpforms fa-2x"></i>
+      <h2 class="org">Forum</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="fal fa-stamp fa-2x"></i>
+      <h2 class="org">Certificate</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="far fa-hands-helping fa-2x"></i>
+      <h2 class="org">Collab</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="far fa-tasks fa-2x"></i>
+      <h2 class="org">Projects</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+  </div>
+    <button>Join LL Coding Together</button>
+    <a class="desktop a-link" href="#">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section class="section-3">
+    <h2>Featured Speakers</h2>
+    <hr>
+    <div class="speaker-container">   
+    </div>
+</section>
+  <section class="section-5">
+    <h2>Partners</h2>
+    <hr>
+    <ul>
+      <li><i class="fab fa-firefox"></i>Mozilla</li>
+      <li><i class="fab fa-google"></i>Google</li>
+      <li><i class="fab fa-free-code-camp"></i>FCC</li>
+      <li><i class="fab fa-microsoft"></i>Microsoft</li>
+      <li><i class="fab fa-airbnb"></i>Airbnb</li>
+    </ul>
+  </section>
+  <footer>
+    <div class="div-1">
+      <i class="footer-icon fas fa-code fa-3x"></i>
+      <h3>Let's Learn Coding Together</h3>
+    </div>
+    <div class="div-2">
+      <p>2021 Let's Code Together.</p>
+      <p>Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
[OPTIONAL ]  Please consider the following design by using the required `background-color` on the `footer' of your home page. This is to make sure that you follow the design guideline and replicate the design to look almost identical to the design guidelines.

 Design guideline: ![fff2](https://user-images.githubusercontent.com/57562869/148335553-b2a3350d-8edc-4e6e-9f6e-66a6a38584dc.png)

 Yours: 
![f1](https://user-images.githubusercontent.com/57562869/148335588-e1b6e99e-08a9-45d4-89fa-ec9a551f1a8a.png)